constructorio_python/constructorio.py

```diff
@@ -0,0 +1,56 @@
+'''ConstructorIO Python Package'''
+
+__version__ = 1.0
+
+from constructorio_python.helpers.exception import ConstructorException
+from constructorio_python.modules.autocomplete import Autocomplete
+from constructorio_python.modules.browse import Browse
+from constructorio_python.modules.recommendations import Recommendations
+from constructorio_python.modules.search import Search
+
+
+class ConstructorIO:
+    '''
+        ConstructorIO Python Client
+
+        :param str api_key: Constructor.io API key
+        :param str api_token: Constructor.io API token
+        :param str security_token: Constructor security token
+        :param str service_url: API URL endpoint
+
+        :return: class
+    '''
+
+    def __init__(self, options) -> None:
+        api_key = options.get('api_key')
+        api_token = options.get('api_token')
+        security_token = options.get('security_token')
+        version = options.get('version')
+        service_url = options.get('service_url')
+        requests = options.get('requests')
+
+        if not api_key or not isinstance(api_key, str):
+            raise ConstructorException('API key is a required parameter of type string')
+
+        self.__options = {
+            'api_key': api_key,
+            'api_token': api_token or '',
+            'security_token': security_token or '',
```

Comment:
You can change `option.get(X)` to `option.get(X, '')` to avoid `or ''` here

Reply:
Oh I didn't know that. Great advice! I will change them